src/actions/entity.js

```diff
@@ -1,4 +1,3 @@
-var INSPECTOR = require('../lib/inspector.js');
```

Comment:
@dmarcos Don't know why but accessing to `INSPECTOR` returns empty object instead of the actual inspector, so I just changed everything here to use `AFRAME.INSPECTOR` but it should works that way. Any idea what could be happening?